bot/exts/evergreen/pythonfacts.py

```diff
@@ -14,20 +15,21 @@
 class PythonFacts(commands.Cog):
     """Sends a random fun fact about Python."""
 
-    def __init__(self, bot: commands.Bot) -> None:
-        self.bot = bot
-
-    @commands.command(name='pythonfact', aliases=['pyfact'])
+    @commands.command(name="pythonfact", aliases=["pyfact"])
     async def get_python_fact(self, ctx: commands.Context) -> None:
         """Sends a Random fun fact about Python."""
-        embed = discord.Embed(title='Python Facts',
-                                    description=next(FACTS),
-                                    colour=next(COLORS))
-        embed.add_field(name='Suggestions',
-                        value="Suggest more facts [here!](https://github.com/python-discord/meta/discussions/93)")
+        embed = discord.Embed(
+            title="Python Facts",
+            description=next(FACTS),
+            colour=next(COLORS)
+        )
+        embed.add_field(
+            name="Suggestions",
+            value="Suggest more facts [here!](https://github.com/python-discord/meta/discussions/93)"
```

Comment:
Hey it's me again with the questionable constants. Should we have this url in a constant?